src/bun.js/api/crypto/EVP.zig

```diff
@@ -36,7 +36,9 @@ pub const Algorithm = enum {
     @"sha3-256",
     @"sha3-384",
     @"sha3-512",
+    @"shake-128",
```

Comment:
these two aren't necessary right? the same enum value as for shake128 and shake256 can be used here